### DISCUSSION TITLE
[Developer Issue] MoorDyn version auto updating

### DISCUSSION THREAD

**Discussion by RyanDavies19** [WEC-Sim/WEC-Sim#1265] [Q&A] [answered]
**Project**
MoorDyn

**Describe the issue**
Hi all, this is in relation to pr #1212 

We just released MoorDyn-C 2.3.0 and will need to update the https://github.com/WEC-Sim/MoorDyn library accordingly. Ideally though we should set up github actions to automate this every time there is a new release. I know @jtgrasb has helped deal with compatibility issues in the past, so some form of testing regime would be good. Throwing this up as an issue so we can discuss potential plans for getting this done.

**Comment by RyanDavies19** [comment]
Also, let me know if there is a better avenue than an issue to go about this.

**Comment by jtgrasb** [comment]
@RyanDavies19 Thank you for making us aware of the new updates. Making our MoorDyn library automatically update is definitely something we are interested in, but also have to be careful if there are any changes that may impact compatibility with WEC-Sim. I am going to move this to a discussion as we work on the automatic updates.

**Comment by RyanDavies19** [comment]
@jtgrasb sounds good, I am concurrently working on auto checking MD-C against OpenFAST as well to make sure everything is staying on the same page. I can begin to chip away at this as I have some time allocated on the NREL side to WEC-Sim

**Comment by jtgrasb** [comment, marked as answer]
Resolved by PR: https://github.com/WEC-Sim/MoorDyn/pull/15
